__integration_tests__/thorchain-client.integration.ts

```diff
@@ -0,0 +1,114 @@
+import { Client as BnbClient } from '@xchainjs/xchain-binance'
+import { Client as BtcClient } from '@xchainjs/xchain-bitcoin'
+import { Client as BchClient } from '@xchainjs/xchain-bitcoincash'
+import { Network, TxParams, XChainClient } from '@xchainjs/xchain-client'
+import { Client as CosmosClient } from '@xchainjs/xchain-cosmos'
+import { Client as EthClient } from '@xchainjs/xchain-ethereum'
+import { Client as LtcClient } from '@xchainjs/xchain-litecoin'
+import { Client as ThorClient, ThorchainClient } from '@xchainjs/xchain-thorchain'
+import {
+  Asset,
+  AssetRuneNative,
+  BaseAmount,
+  Chain,
+  assetFromString,
+  assetToString,
+  baseAmount,
+} from '@xchainjs/xchain-util'
+
+export type Swap = {
+  fromBaseAmount: BaseAmount
+  to: Asset
+}
+
+let xchainClients: Record<Chain, XChainClient>
```

Comment:
Not needed ^ , since all of e2e tests in this file are for `xchain-thorchain` only. I would recommend to move `e2e` tests into `packages/xchain-thorchain/__tests__/e2e/transaction.test.ts`